src/components/Expander/Expander.less

```diff
@@ -67,4 +67,35 @@
 	&-content {
 		overflow: hidden;
 	}
+
+	// kind
+	&-with-emphasis {
+
+		.lucid-Expander-header {
+			background: @color-backgroundColor;
+			border: 1px solid @color-borderColorLight;
+			padding: 12px;
+			display: flex;
+
+			.lucid-Expander-text {
+				font-size: 15px;
+				color: @color-textColor;
+				font-weight: bold;
+				flex: 1;
+			}
+
+			.lucid-Expander-icon {
+				margin: 0;
+				flex-shrink: 0;
+			}
+
+			&:hover .lucid-Expander-text > span {
+				text-decoration: none;
+			}
+		}
+
+		.lucid-Collapsible-content {
+			padding: 10px;
```

Comment:
I think this should be `@size-standard` as well to match the styling of `ExpanderPanel` and to align with the header.